PIM/Operation/01_ManageProducts.md

```diff
@@ -0,0 +1,304 @@
+[!!Omni-Channel](Channels)   
+[!!User Interface PIM](../UserInterface/00_UserInterface.md)   
+[!!Manage the variants](./02_ManageVariants.md)
+
+
+# Manage a product
+
+A product defines everything that can be offered and purchased on a market. All products are assigned to an attribute set and can be specified by the attributes within this set. You can create master products and [variants](./02_ManageVariants.md#create-variants), edit and copy products. To offer a product on a marketplace you have to create an offer to the product, see [Create an offer in PIM](#create-an-offer-in-PIM).
+
+## Create a product
+
+Create a product to offer it on a marketplace. You can create a product or create variants, see [Create variants](./02_ManageVariants.md#create-variants). In the following, the procedure to create a product is described.
+
+#### Prerequisites
+
+An attribute set is created, see [Create an attribute set](../Integration/02_ManageAttributeSets.md#Create-an-attribute-set).
```

Comment:
An Attribute set does not have to be created, you can in theory also use the PIM Base set. But it is recommended to create AttributeSets in order to avoid having to many unnecessary Attributes in the Attribute set.